src/main/java/kitchenpos/order/domain/Order.java

```diff
@@ -0,0 +1,103 @@
+package kitchenpos.order.domain;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long orderTableId;
+    private String orderStatus;
+    private LocalDateTime orderedTime;
+    @Embedded
+    private OrderLineItems orderLineItems;
+
+    public Order() {
+    }
+
+    public Order(Long orderTableId) {
+        this.orderTableId = orderTableId;
+        this.orderStatus = OrderStatus.COOKING.name();
+        this.orderedTime = LocalDateTime.now();
+    }
+
+    public Order(Long orderTableId, List<OrderLineItem> orderLineItems) {
+        this.orderTableId = orderTableId;
+        this.orderLineItems = new OrderLineItems(orderLineItems);
+        this.orderStatus = OrderStatus.COOKING.name();
+        this.orderedTime = LocalDateTime.now();
+    }
+
+    public Order(String orderStatus) {
+        this.orderStatus = orderStatus;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(final Long id) {
+        this.id = id;
+    }
+
+    public Long getOrderTableId() {
+        return orderTableId;
+    }
+
+    public String getOrderStatus() {
+        return orderStatus;
+    }
+
+    public void changeOrderStatus(final String orderStatus) {
+        this.orderStatus = orderStatus;
+    }
+
+    public LocalDateTime getOrderedTime() {
+        return orderedTime;
+    }
+
+    public List<OrderLineItem> getOrderLineItems() {
+        return orderLineItems.getOrderLineItems();
+    }
+
+    public void addItems(List<OrderLineItem> items) {
+        this.orderLineItems = new OrderLineItems(items);
+    }
+
+    public boolean statusIsMeal() {
```

Comment:
보통 boolean을 반환하는 메서드명은 is로 시작하는게 관례입니다.

```suggestion
    public boolean isMeal() {
```